### PR TITLE
🚧(service engineer)- add models, events and handlers

### PR DESCRIPTION
**Pull request by thangeniFT**
not yet functional, simply models, events and events handlers and declaration of constructors

### CODE REVIEW

File: libs/api/upintegration/service/feature/src/lib/commands/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+//import { getStudentInfoCommand } from "./getStudentInfo-upintegration.commands";
+
+//export {getStudentInfoCommand};
```
Comment (u20437863-James-Butler) [inline libs/api/upintegration/service/feature/src/lib/commands/index.ts:3]: Looks good to me, can you just clarify what was commented out in this file? I want to make sure its not important.

Comment (thangeniFT) [reply libs/api/upintegration/service/feature/src/lib/commands/index.ts:3]: it's of no importance as of yet

Comment (u20437863-James-Butler) [reply libs/api/upintegration/service/feature/src/lib/commands/index.ts:3]: Ok, will merge then

**Review by u20437863-James-Butler — APPROVED** [review]
All the files look good

**Pull request merged**